### ISSUE TITLE
External Sequence Analysis - PFAM 

### ISSUE THREAD

**Issue by cfcgoncalves** [kvittingseerup/IsoformSwitchAnalyzeR#75]
Hello,

I've been having trouble adding PFAM results to my data. 

```
#Add PFAM analysis
aSwitchList1 <- analyzePFAM(
    switchAnalyzeRlist   = aSwitchList1,
    pathToPFAMresultFile = "/Users/filip/Desktop/PFAM/list1_1.txt",
    showProgress=FALSE
)
```

Which gives the following error:

```
Converting AA coordinats to transcript and genomic coordinats...
Error in myPfamResult$orf_aa_end * 3 : 
  non-numeric argument to binary operator
```

However, I don't get the same error for all txt files I have. For example:

```
#Add PFAM analysis
aSwitchList1 <- analyzePFAM(
    switchAnalyzeRlist   = aSwitchList1,
    pathToPFAMresultFile = "/Users/filip/Desktop/PFAM/list1_5.txt",
    showProgress=FALSE
)
```
```
Converting AA coordinats to transcript and genomic coordinats...
Added domain information to 96 (0.55%) transcripts
```

I can't figure out what's the difference between the files as they were copied and saved in exactly the same way. All the other external sequence analysis go through normally (CPAT, IUPred2A, SignalIP).

Any suggestions appreciated. 
Thank you very much!

**Comment by kvittingseerup** [comment]
Hi 

Thanks for reaching out.

I think this will be solved by updating to IsoformSwitchAnalyzeR 1.11.3 as described [here](https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html).

Let me know if it works.

Cheers
Kristoffer

**Comment by cfcgoncalves** [comment]
Hi Kristoffer,

It worked! Thanks a lot for your help. 

Best wishes,
Cátia
